src/utils/index.ts

```diff
@@ -70,3 +70,29 @@ export const sortByProperty =
       return 0;
     }
   };
+
+export const sortByNestedProperty =
```

Comment:
The previous `sortByProperty` is not yet widely used, so I'd replace it with this function.

Reply:
replaced sortByProperty implementation with this function.